tools/refguide_check.py

```diff
@@ -24,21 +24,23 @@
 """
 from __future__ import print_function
 
-import sys
-import os
-import re
 import copy
-import inspect
-import warnings
 import doctest
-import tempfile
+import glob
+import inspect
 import io
-import docutils.core
-from docutils.parsers.rst import directives
+import os
+import re
 import shutil
-import glob
-from doctest import NORMALIZE_WHITESPACE, ELLIPSIS, IGNORE_EXCEPTION_DETAIL
+import sys
+import tempfile
+import warnings
+import docutils.core
 from argparse import ArgumentParser
+from contextlib import contextmanager, redirect_stderr
+from doctest import NORMALIZE_WHITESPACE, ELLIPSIS, IGNORE_EXCEPTION_DETAIL
+
+from docutils.parsers.rst import directives
```

Comment:
Why the change in the import ordering? 
The reordering is not called out in the commit or PR description, so asking about the rationale for it.  [I wondered if it might have been making the imports PEP8-compliant but that doesn't seem to have been the case.]

Reply:
I was trying to make it closer to PEP8-compliant, I've reordered them to be what I believe is PEP8 compliant (I'm not sure how to sort the `from numpydoc.docscrape_sphinx import` which requires modifying sys.path) and to lead comment in this PR.

Reply:
Given that the `from numpydoc.docscrape_sphinx import ...` has to occur after the `sys.path` change, there is really no other place it can go.